apps/src/lib/node/ledger/mod.rs

```diff
@@ -44,6 +44,10 @@ const ENV_VAR_TOKIO_THREADS: &str = "ANOMA_TOKIO_THREADS";
 /// Env. var to set a number of Rayon global worker threads
 const ENV_VAR_RAYON_THREADS: &str = "ANOMA_RAYON_THREADS";
 
+/// The maximum number of Ethereum events the channel between
+/// the oracle and the shell can hold.
+const ORACLE_CHANNEL_BUFFER_SIZE: usize = 1000;
```

Comment:
This const should live elsewhere or even be a `ledger.ethereum` config option (in `apps/src/lib/config/ethereum.rs`), but I think it's fine to leave it as is in this PR and make that change separately

Reply:
Sounds good, will make an issue.